package.json

```diff
@@ -6,14 +6,14 @@
     "start": "node app.js",
     "console": "node scripts/console.js",
     "eslint": "./node_modules/.bin/grunt eslint-nofix --verbose",
-    "fetchCoverage": "./node_modules/.bin/grunt exec:fetchCoverage --verbose",
-    "coverageReport": "./node_modules/.bin/grunt coverageReport --verbose",
+    "cover:fetch": "./node_modules/.bin/grunt exec:fetchCoverage --verbose",
+    "cover:report": "./node_modules/.bin/grunt coverageReport --verbose",
     "test": "./node_modules/.bin/grunt",
     "grunt": "./node_modules/.bin/grunt",
     "mocha": "./node_modules/.bin/mocha",
     "jsdoc": "jsdoc -c docs/conf.json --verbose --pedantic",
-    "create-bundles": "webpack",
-    "server-docs": "npm run jsdoc && http-server docs/jsdoc/",
+    "build": "webpack",
+    "build:docs": "npm run jsdoc && http-server docs/jsdoc/",
```

Comment:
I think the important thing here is that the docs get served, not that they're built. The build script is what's currently named `jsdoc`.